### ISSUE TITLE
MSO tag breaks replying and forwarding

### ISSUE THREAD

**Issue by radiKal07** [k9mail/k-9#2848]
There are some mails that contain a mso tag like so:

```
<!--[if mso]>
  <body class="mso"><div class="gmail_quote"><br><br><br><blockquote class="gmail_quote" style="margin: 0pt 0pt 0pt 0.8ex; border-left: 1px solid rgb(204, 204, 204); padding-left: 1ex;">
...
<![endif]-->
```

If you reply or forward this mail and you add some more text, this new text will not be visible/

### Expected behavior
1. Read an email containing mso tags.
2. Reply to it and add extra text
3. Open the new email and see the content plus the extra added text from the reply.

### Actual behavior
1. Read an email containing mso tags.
2. Reply to it and add extra text
3. Open the new email and see the original content but the extra text from the reply is not visible because the extra text is included in the commented block.

### Steps to reproduce
1. Receive an email with mso tags
2. Reply to it
3. The new text is appended to the commented block with the mso tags so it will not be visible.

### Environment
K-9 Mail version: 5.208

Android version: Android 7

Account type (IMAP, POP3, WebDAV/Exchange): IMAP

**Comment by cketti** [comment]
Duplicate of #2148

**Comment by radiKal07** [comment]
@cketti 
#2148 is closed.
This issue is still reproducing.
Why close it ?

**Comment by cketti** [comment]
This is an issue tracker for the development of the app. We close issues as soon as they are fixed in the source code repository.
It usually takes a while for fixes to show up in a stable release. This one is included in K-9 Mail v5.301 (which is an alpha release, but should be pretty stable).
